kore/src/Kore/Step/SMT/Evaluator.hs

```diff
@@ -195,4 +204,10 @@ translateUninterpreted t pat =
         n <- Counter.increment
         var <- SMT.declare ("<" <> Text.pack (show n) <> ">") t
         State.modify' (Map.insert pat (var, t))
+        Monad.Trans.lift . Monad.Trans.lift . Monad.Trans.lift $
```

Comment:
Why were all these `lift`s necessary?

Reply:
I' not quite sure... the program won't compile until I added three `lift`s...I looked at the definition of Translator:
```
type Translator m p = MaybeT (StateT (VarContext p) (CounterT m))
``` and Monad `m` is transformed three times. Maybe this is the reason?